config/environments/development.rb

```diff
@@ -21,6 +21,16 @@
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
+  config.paperclip_defaults = {
+    :storage => :s3,
+    :s3_host_name => 's3-us-west-1.amazonaws.com',
+    :s3_credentials => {
+      :bucket => ENV['S3_BUCKET_NAME'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.